src/lib/db/group-store.ts

```diff
@@ -60,10 +60,14 @@ const groupToRow = (group: IStoreGroup) => ({
     mappings_sso: JSON.stringify(group.mappingsSSO),
 });
 
-export default class GroupStore implements IGroupStore {
-    private db: Knex;
+export default class GroupStore
+    extends Transactor<GroupStore>
+    implements IGroupStore
+{
+    private db: Knex | Knex.Transaction;
 
     constructor(db: Knex) {
+        super();
```

Comment:
Should this be: 
```suggestion
        super(db);
```

I'm guessing because https://github.com/Unleash/unleash/pull/2217/files#diff-d1c23d22abb0f437d1416e15b6351a785373db023f45dd51090027c060582c93R51 and in most cases we're matching the constructor

Reply:
Yes, good eyes, thank you! It's not _strictly_ necessary for a store that only contains the db object, the clone will still work and the TS compiler will allow it but it's good to be clear